src/api/cow/cow.spec.ts

```diff
@@ -693,3 +703,63 @@ test('Valid: Update sdk context', async () => {
   await cowSdk.updateContext({ env: 'prod' })
   expect(cowSdk.context.env).toEqual('prod')
 })
+
+describe('Transform EthFlow orders', () => {
+  test('getOrder', async () => {
+    fetchMock.mockResponseOnce(JSON.stringify(ETH_FLOW_ORDER_RESPONSE), {
+      status: HTTP_STATUS_OK,
+      headers: HEADERS,
+    })
+
+    // when
+    const order = await cowSdk.cowApi.getOrder(ETH_FLOW_ORDER_RESPONSE.uid)
+
+    // then
+    expect(order?.owner).toEqual(order?.onchainUser)
+    expect(order?.validTo).toEqual(order?.ethflowData?.userValidTo)
+    expect(order?.sellToken).toEqual(BUY_ETH_ADDRESS)
```

Comment:
wouldn't we want to have the `ownerOriginal` or sth, and assert is there?

Reply:
The original owner is being overwritten, as well as the original validTo.
Do you think we should keep them?